tests/benchmarks/multicore/idle/src/main.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(idle);
+
+int main(void)
+{
+	int cnt = 0;
+
+	LOG_INF("Multicore idle test on %s\n", CONFIG_BOARD_TARGET);
+	while (1) {
+		LOG_INF("Multicore idle test iteration %d\n", cnt++);
```

Comment:
Do we need `\n` here? The effect is double interline.

Reply:
Fixed